server/app/views/admin/questions/QuestionsListView.java

```diff
@@ -158,9 +160,21 @@ private static QuestionDefinition getDisplayQuestion(QuestionCardData cardData)
     return cardData.draftQuestion().orElseGet(cardData.activeQuestion()::get);
   }
 
+  private static boolean questionIsPendingDeletion(
+      QuestionCardData card, ActiveAndDraftQuestions activeAndDraftQuestions) {
+    return questionIsPendingDeletion(getDisplayQuestion(card), activeAndDraftQuestions);
+  }
+
+  private static boolean questionIsPendingDeletion(
```

Comment:
Nit- May be rename this to isQuestionPendingDeletion() as we are returning a boolean?

Reply:
Done